amulet/sentry.py

```diff
@@ -120,10 +120,11 @@ def upload_scripts(self):
 
         # copy one at a time b/c `juju scp -r` doesn't work (currently)
         for f in glob.glob(os.path.join(source, '*')):
-            cmd = "juju scp {} {}:{}".format(
-                os.path.join(source, f),
-                self.info['unit_name'], dest)
-            subprocess.check_call(cmd.split())
+            src = os.path.join(source, f)
+            if os.path.isfile(src):
```

Comment:
See the stack trace in the linked issue.  It's picking up `__pycache__` directories.  Changing the glob is a valid alternative.